src/redux/reducer.js

```diff
@@ -174,10 +174,18 @@ export const reducer = createReducer(initialState, {
         if (state.networkModificationTreeModel) {
             let newModel =
                 state.networkModificationTreeModel.newSharedForUpdate();
+            let parentNode;
+            if (
+                action.networkModificationTreeNodes.includes(
+                    state.currentTreeNode?.id
+                )
+            ) {
+                parentNode = state.currentTreeNode?.data?.parentNodeUuid;
```

Comment:
Add a TODO comment here :
Today we manage action.networkModificationTreeNodes which size is always 1 and then to delete one node at a time.
If tomorrow we need to delete multiple nodes, we need to check that the parentNode here isn't in the action.networkModificationTreeNodes list

Reply:
Ok, done.